meshroom/nodes/aliceVision/ApplyCalibration.py

```diff
@@ -27,6 +27,13 @@ class ApplyCalibration(desc.AVCommandLineNode):
             value="",
             uid=[0],
         ),
+        desc.BoolParam(
+            name="useJson",
+            label="use Json",
+            description="Use generated json file instead of sfmData file.",
+            value=False,
+            uid=[0],
+        ),
```

Comment:
Maybe easier to understand to have a Choice "Calibration File Type" with "SfmData" or "Lens Calibration" .
Or we could keep it implicit and rely on the file extension (.sfm/.abc and .json).